scripts/skills/perks/perk_rf_feral_rage.nut

```diff
@@ -111,50 +83,28 @@ this.perk_rf_feral_rage <- ::inherit("scripts/skills/skill", {
 
 	function onUpdate( _properties )
 	{
-		_properties.DamageReceivedTotalMult *= ::Math.maxf(0.3, 1.0 - 0.02 * this.m.RageStacks);
-		_properties.Bravery += this.m.RageStacks;		
-		_properties.Initiative += this.m.RageStacks;
+		_properties.DamageReceivedTotalMult *= ::Math.maxf(0.3, 1.0 - 0.03 * this.m.RageStacks);
```

Comment:
Should use the `getDamageReceivedMult()` utility function proposed above

Reply:
Instead of a function I've just extracted the numbers into member variables and copied the equation in both tooltip and onUpdate function because the equation is very straightforward. Are you ok with this?